src/util/aux_util.h

```diff
@@ -84,6 +89,45 @@ extern "C" {
         r_max = r; \
     }
 
+static inline TSS2_RC
```

Comment:
Since this is only needed in Esys_TestParms.c maybe it should be defined there?

Reply:
Thats fine, I struggled with that decision myself. I figured though, that this is generally usable. But I have no strong opinion, so Ill just move it over. We can always move it again if needed.

Reply:
Most routines like this seem to be not in the API implementation folders. The only static routines I see are the store_input routines. Which also have a slightly different style in that the return type and function name are on one line rather than down another level...